debezium-core/src/main/java/io/debezium/relational/RelationalChangeRecordEmitter.java

```diff
@@ -33,8 +34,8 @@
     public static final String PK_UPDATE_OLDKEY_FIELD = "__debezium.oldkey";
     public static final String PK_UPDATE_NEWKEY_FIELD = "__debezium.newkey";
 
-    public RelationalChangeRecordEmitter(P partition, OffsetContext offsetContext, Clock clock) {
-        super(partition, offsetContext, clock);
+    public RelationalChangeRecordEmitter(P partition, OffsetContext offsetContext, Clock clock, CommonConnectorConfig connectorConfig) {
```

Comment:
@ironakj Can we use `RelationalDatabaseConnectorConfig` here?

Reply:
Done